docs/install/kubernetes.md

```diff
@@ -3,149 +3,143 @@ title: "Kubernetes"
+Verify that the Local Path Provisioner was successfully deployed and ensure that that the deployment is in a  `RUNNING` status, run the following `kubectl` commands.
 
-#### Step 2 - Conducting Preflight Checks
+```bash
+kubectl get pod --namespace=local-path-storage
+kubectl get storageclass
+```
 
-* Run the following command to conduct preflight checks against the Kubernetes cluster to validate that Ondat prerequisites have been met before attempting an installation.
+## Installation of Ondat
 
-    ```bash
-    kubectl storageos preflight
-    ```
+### Step 1 - Adding a Cluster
 
-#### Step 3 - Installing Ondat
+The Ondat Portal is how you can license and get the commands for installing Ondat
 
-1. Define and export the `STORAGEOS_USERNAME` and `STORAGEOS_PASSWORD` environment variables that will be used to manage your Ondat instance.
+* Either login or create an account on the [Ondat Portal](https://portal.ondat.io/)
+* Choose the 'Install Ondat on your cluster' or 'Add cluster' options in the UI
+* Add a Name for your cluster and where it is going to be located.  This will allow you to view the same prerequisites as are listed above
 
-    ```bash
-    export STORAGEOS_USERNAME="storageos"
-    export STORAGEOS_PASSWORD="storageos"
-    ```
+### Step 2 - Choosing the Installation Method
 
-1. Run the following  `kubectl-storageos` plugin command to install Ondat.
+You can use either the [kubectl-storageos CLI](/docs/reference/kubectl-plugin/) or [Helm 3 CLI](https://helm.sh/docs/intro/install/) to install Ondat onto your cluster.  The most common way is to use Helm due to its popularity in the Kubernetes community, but both are fully supported and described below.
 
-    ```bash
-    kubectl storageos install \
-      --include-etcd \
-      --etcd-tls-enabled \
-      --etcd-storage-class="$ETCD_STORAGECLASS" \
-      --admin-username="$STORAGEOS_USERNAME" \
-      --admin-password="$STORAGEOS_PASSWORD"
-    ```
+### Step 3a - Installing via Helm
 
-* The installation process may take a few minutes.
+The Ondat Portal UI will display the following cmd that can be used to install Ondat using Helm.
 
-#### Step 4 - Verifying Ondat Installation
+![Helm Install](/images/docs/install/HelmInstall.png)
 
-* Run the following `kubectl` commands to inspect Ondat's resources (the core components should all be in a `RUNNING` status)
+1. The first set of commands adds the Ondat Helm repository and ensures a updated local cache.
 
-    ```bash
-    kubectl get all --namespace=storageos
-    kubectl get all --namespace=storageos-etcd
-    kubectl get storageclasses | grep "storageos"
-    ```
+```bash
+helm repo add ondat https://ondat.github.io/charts && \
+helm repo update && \
+```
 
-### Option B - Using An External `etcd` Deployment
+2. The last command installs Ondat with a set of basic install parameters that are sufficient for a basic trial installation.
 
-#### Step 1 - Setup An `etcd` Cluster
+```bash
+helm install ondat ondat/ondat \
```

Comment:
We are missing the etcd storage class, or a mention that a default StorageClass needs to exist.

Reply:
The installation of the StorageClass is added to Prerequisites Section 3 - Installing a Local Path Provisioner